pkg/scheduler/framework/plugins/podtopologyspread/scoring.go

```diff
@@ -219,9 +220,10 @@ func (pl *PodTopologySpread) NormalizeScore(ctx context.Context, cycleState *fra
 	// Calculate <minScore> and <maxScore>
 	var minScore int64 = math.MaxInt64
 	var maxScore int64
-	for _, score := range scores {
+	for i, score := range scores {
```

Comment:
doing `for i, score := range scores` would make a copy of score for each element -- does `for i := range score` do better? kinda premature optimisation, but worth a quick check :)

Reply:
The object is so small that it doesn't really matter.